### PR TITLE
being more explicit for zone limitation

### PR DESCRIPTION
**Pull request by l0k0ms**

### CODE REVIEW

File: content/integrations/dyn.md [hunk 1]
```diff
@@ -34,9 +33,10 @@ Once you have created a Datadog Read only user:
 4. Configure your [Dyn integration](https://app.datadoghq.com/account/settings#integrations/dyn) inside your Datadog application.
 {{< img src="integrations/dyn/dyn_integration.png" alt="Dyn Integration" style="width:75%;" >}}
 
-5. Select your zones you want to monitor:
-{{< img src="integrations/dyn/dyn_zone.png" alt="Dyn zone" style="width:50%;" >}}
-
+5. Select your zones you want to monitor:<br>
```
Comment (irabinovitch) [inline content/integrations/dyn.md]: Select the zones you wish to monitor

File: content/integrations/dyn.md [hunk 2]
```diff
@@ -34,9 +33,10 @@ Once you have created a Datadog Read only user:
...
+{{< img src="integrations/dyn/dyn_zone.png" alt="Dyn zone" style="width:75%;" >}}
+The “Zone limitation” applies only to “Zone notes”.
+**Dyn “QPS” metrics are by default collected for all zones**. It's Events and Dyn changes metrics that aren’t.
```
Comment (irabinovitch) [inline content/integrations/dyn.md]: Minor edit: `**Dyn “QPS” metrics are by default collected for all zones**, however events generated by Dyn changes are not.`

Are we saying one needs to select the zones to collect events from? What do we mean by Dyn changes?

Wht is the limitation? # of zones to monitor?

Comment (kmshultz) [reply content/integrations/dyn.md]: > It's Events and Dyn changes metrics that aren’t.

Events and Dyn changes are not.

File: content/integrations/dyn.md (hunk 1 above)
Comment (l0k0ms) [reply content/integrations/dyn.md]: :+1:

**Pull request closed without merging**